src/vanilla/utils.ts

```diff
@@ -12,5 +12,5 @@ export {
   createJSONStorage,
 } from './utils/atomWithStorage'
 export { atomWithObservable } from './utils/atomWithObservable'
-export { loadable } from './utils/loadable'
+export { loadable, Loadable } from './utils/loadable'
```

Comment:
Let's not export from the index. (Like  `AtomFamily`.)
```suggestion
export { loadable } from './utils/loadable'
```